src/test/java/Homework19.java

```diff
@@ -0,0 +1,30 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+        String deletedPlaylistMsg = "Delete playlist";
+
+        logIn("demo@class.com","te$t$tudent");
+        openPlaylist();
+        clickDeletePlaylistBtn();
+        Assert.assertTrue(getDeletedPlaylistMsg().contains(deletedPlaylistMsg));
+    }
+    public void openPlaylist() {
+        WebElement emptyPlaylist = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector(".playlist:nth-child(3)")));
+        emptyPlaylist.click();
+    }
+    public void clickDeletePlaylistBtn() {
+        WebElement deletePlaylist = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector(".btn-delete-playlist")));
+        deletePlaylist.click();
+    }
+    public String getDeletedPlaylistMsg() {
+        WebElement notificationMsg = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("div.success.show")));
+        return notificationMsg.getText();
+    }
+}
```

Comment:
Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/226269389-af9e9d10-5268-48e0-b286-b90083e1a330.png)

HINT

![image](https://user-images.githubusercontent.com/104761482/226269654-3f72acc1-dbd4-4c32-9e9a-ca9bf9a543cf.png)

![image](https://user-images.githubusercontent.com/104761482/226269780-4b1f3543-7ee6-4861-80b1-8ea15b7f8422.png)